btsieve/src/ethereum/queries/transaction.rs

```diff
@@ -18,46 +18,42 @@ use serde::{Deserialize, Serialize};
 pub struct TransactionQuery {
     from_address: Option<Address>,
     to_address: Option<Address>,
-    is_contract_creation: Option<bool>,
```

Comment:
Why?

Reply:
@tcharding 

This change is semantically incorrect :)
We need three states: None, true and false.

With the proposed change, we cannot express a query where we "don't care" about whether it is a contract creation or not.